test/fixtures/output/network/network-link/service/create-accepted-environments.golden

```diff
@@ -0,0 +1,9 @@
++-----------------------+----------------------+
+| ID                    | nls-abcde1           |
+| Name                  | nls-test             |
```

Comment:
Let's change this to something that looks more like a name than an ID, for example "my-network-link-service".